app/helpers/waste_carriers_engine/application_helper.rb

```diff
@@ -60,11 +60,6 @@ def displayable_address(address)
        address.country].reject(&:blank?)
     end
 
-    # WickedPdf does come with helpers that you can use
```

Comment:
I'd be happy to drop this if we were only dealing with certificates for emails. However we also allow certificates to be viewed as PDFs in the back office and front office – see https://github.com/DEFRA/waste-carriers-back-office/blob/main/app/controllers/certificates_controller.rb for an example – so might be worth confirming how this change (and adding a `@presenter` variable in the template) will affect it there.

Reply:
Removing the wicked_pdf helper:
- I spent some time trying to mix in the ApplicationHelper into the service so `wicked_pdf_image_tag` was accessible from the template. Then I realised that the service doesn't actually use wicked_pdf so it felt wrong to load all the helpers for this one method was just formatting an image tag. 
- Have checked with the front office app and the PDF certificate is generated as expected, with the logo.

`@presenter` variable:
- The template already used the instance variable. Have updated the comment on the certificate template

Reply:
Aha, that makes sense! Thanks for clarifying :)